proposals/inline-classes.md

```diff
@@ -38,12 +38,57 @@ inline class ULong(private val value: Long) { ... }
 - Units of measurement
 - Result type (aka Try monad) [KT-18608](https://youtrack.jetbrains.com/issue/KT-18608)
 - Inline property delegates
+```kotlin
+class A {
+    var something by InlinedDelegate(Foo()) // no actual instantiation of `InlinedDelegate`
+}
+
+
+inline class InlinedDelegate<T>(var node: T) {
```

Comment:
> Inline class must have a single read-only (val) property as an underlying value, which is defined in primary constructor

There is an inconsistency

Reply:
I'd say that this is an open question. We can support mutable inline classes if they will be used only with delegated properties. Currently we're looking for use cases

Reply:
Hi @zarechenskiy,
having a delegate with a backing field is interesting, however rules exceptions are harder to explain and understand.